File: Patika.dev Coderbyte Assessments (Weekly)/Week 5/Java/1 - Power Set Count/PowerSetCount.java
import java.util.*;
import java.io.*;

class PowerSetCount {

    public static int powerSetCount(int[] arr) {
        // code goes here
        return (int) Math.pow(2, arr.length);
    }

    public static void main (String[] args) {
        // keep this function call here
        Scanner s = new Scanner(System.in);
        System.out.print(powerSetCount(s.nextLine()));
    }

}

File: Easy/Power Set Count.py
from itertools import chain,combinations

def PowerSetCount(arr):

    # code goes here
    try:
        power_set = list(chain(*map(lambda x: combinations(arr, x), range(0, len(arr) + 1))))
        return len(power_set)
    except(AttributeError, TypeError):
        return -1

# keep this function call here
print(PowerSetCount(input()))